airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -199,6 +201,8 @@ class Stream(ABC):
     limit_field = "limit"
     limit = 100
     offset = 0
+    _name: str = None
```

Comment:
This property is added to define stream name when use CRMSearchStream, CRMObjectIncrementalStream general (template) stream. So self._name = "companies"  matches to  `schema/companies.json`